Tests/test_NCBI_qblast.py

```diff
@@ -48,7 +48,7 @@ def test_blastp_nr_actin(self):
         # GI:160837788 aka NP_075631.2
         # the actin related protein 2/3 complex, subunit 1B [Mus musculus]
         self.run_qblast("blastp", "nr", "NP_075631.2", 0.001,
-                        "rat [ORGN]", {'megablast': 'FALSE'},
```

Comment:
Does the old test still work? If so can we keep the old test using entrez_query, and add a similar new test using the organism field?

Reply:
Yes, it works. I revised this part to prove two methods could get the same results, so they are equivalent. Of course I could add a similar test.